### DISCUSSION TITLE
How can I filter nodes by host label?

### DISCUSSION THREAD

**Discussion by shadycuz** [netdata/netdata#15620] [Q&A]
Hello =), 

I have recently convinced my work to do a POC of Netdata. Mainly because of the new parent/child feature. 

I currently have it working between 1 parent and 2 children.  I'm struggling to find a way to filter these nodes. I would like to filter by the labels automatically added by netdata and by custom host labels but I can't seem to find a way to do it. When I search for documentation, I only find [how to add labels](https://learn.netdata.cloud/docs/configuring/organize-systems-metrics-and-alerts#host-labels) but not what I can do with them. 

I did find this old feature request https://github.com/netdata/netdata/issues/7095. With the new Parent/Child feature, I think its really important to be able to select groups of infra by label. Seeing the CPU across 150 nodes where most of the nodes are not related is not very helpful. Seeing the CPU across 20 nodes that are related is very helpful.

**Comment by hugovalente-pm** [comment]
Hi @shadycuz,

Thank you for opening this question, you're totally right the filtering based on labels is something we want to bring into Netdata capabilities.
We have an internal ticket about this that I believe it is in a state that it can be moved to public (will aim to that) and you and other users can contribute on the discussions.

We haven't been able to tackle this yet because of other urgent priorities we had in the roadmap.

I know it isn't mentioned on the [What's New and Coming?](https://github.com/netdata/netdata#whats-new-and-coming) section of our readme but it certainly is related somehow to Easy Custom Dashboards.

Basically just sharing that is something we want to bring into Netdata and hopefully we can work on it soon.

P.S.: Nice yo know you've managed to start a POC of Netdata, if there is anything we can help with let us know.
